pkg/volume/util/attach_limit.go

```diff
@@ -26,4 +31,25 @@ const (
 	AzureVolumeLimitKey = "attachable-volumes-azure-disk"
 	// GCEVolumeLimitKey stores resource name that will store volume limits for GCE node
 	GCEVolumeLimitKey = "attachable-volumes-gce-pd"
```

Comment:
Do the in-tree limit keys need to be changed to be compatible with the csi driver equivalent?

Reply:
You mean - once migration happens? I think we will stick to the plan of carrying around a map for in-tree plugins when we start shipping those.

Reply:
Yes, I mean, now that this is going beta, we need to support these limits keys for two releases.  So there will be a period of time where we'll need some in-tree volume key to csi translation function